Documentation/SwiftGenKit Contexts/Strings.md

```diff
@@ -2,7 +2,9 @@
 
 ## Input
 
-The strings parser accepts a `strings` file, typically `Localizable.strings`. It will parse each string in this file, including the type information for formatting parameters. 
+The strings parser accepts a `strings` file, typically `Localizable.strings`, or a directory.  
+In case it is passed a file it will parse each string in this file, including the type information for formatting parameters.  
+In the case of a directory it will look recursively in the directory and parse each `strings` file it finds. Bear in mind that only one language can be used to generate the `enum` so do not pass a directory that includes multiple localizations of the same `strings` file.
```

Comment:
s/can be used to generate the enum/should be passed (to avoid parsing the same keys multiple times in different languages)/

s/multiple localizations of/multiple `.lproj` folders with/

Reply:
ok